Client/game_sa/CSettingsSA.cpp

```diff
@@ -565,39 +565,24 @@ void CSettingsSA::SetFieldOfViewVehicleMax(float fAngle, bool bFromScript)
 ////////////////////////////////////////////////
 float ms_fClientMaxVehicleLODDistance = DEFAULT_VEHICLE_LOD_DISTANCE;
 float ms_fClientMaxTrainPlaneLODDistance = DEFAULT_VEHICLE_LOD_DISTANCE * TRAIN_LOD_DISTANCE_MULTIPLIER;
-float ms_fScriptMaxVehicleLODDistance = ms_fClientMaxVehicleLODDistance;
-float ms_fScriptMaxTrainPlaneLODDistance = ms_fClientMaxTrainPlaneLODDistance;
 bool  ms_bMaxVehicleLODDistanceFromScript = false;
 
 void CSettingsSA::SetVehiclesLODDistance(float fVehiclesLODDistance, float fTrainsPlanesLODDistance, bool bFromScript)
 {
-    if (bFromScript)
-    {
-        ms_fScriptMaxVehicleLODDistance = fVehiclesLODDistance;
-        ms_fScriptMaxTrainPlaneLODDistance = fTrainsPlanesLODDistance;
-        ms_bMaxVehicleLODDistanceFromScript = bFromScript;
-    }
-    else
+    if (!bFromScript)
     {
         ms_fClientMaxVehicleLODDistance = fVehiclesLODDistance;
         ms_fClientMaxTrainPlaneLODDistance = fTrainsPlanesLODDistance;
     }
 
-    if (ms_bMaxVehicleLODDistanceFromScript)
-    {
-        ms_fVehicleLODDistance = Min(ms_fClientMaxVehicleLODDistance, ms_fScriptMaxVehicleLODDistance);
-        ms_fTrainPlaneLODDistance = Min(ms_fClientMaxTrainPlaneLODDistance, ms_fScriptMaxTrainPlaneLODDistance);
-    }
-    else
-    {
-        ms_fVehicleLODDistance = Min(fVehiclesLODDistance, ms_fClientMaxVehicleLODDistance);
-        ms_fTrainPlaneLODDistance = Min(fTrainsPlanesLODDistance, ms_fClientMaxTrainPlaneLODDistance);
-    }
+    ms_fVehicleLODDistance = fVehiclesLODDistance;
+    ms_fTrainPlaneLODDistance = fTrainsPlanesLODDistance;
+    ms_bMaxVehicleLODDistanceFromScript = bFromScript;
 }
 
-void CSettingsSA::ResetVehiclesLODDistance(bool bFromScript)
+void CSettingsSA::ResetVehiclesLODDistance(bool bFromScript, bool bFromConfig)
```

Comment:
Why this method looks so complicated? Couldn't it look more like `CSettingsSA::ResetCoronaReflectionsEnabled()`?